docs/guides/migration_guide_v2.md

```diff
@@ -66,11 +69,57 @@ Below you find an overview of the changes in environment variables:
 | `SCW_TOKEN`             | `SCW_SECRET_KEY`                            |
 
-~> **Important:** `SCALEWAY_ACCESS_KEY` was changed to `SCW_ACCESS_KEY`. This should be your access key (e.g. `SCWZFD9BPQ4TZ14SM1YS`). Your secret key (previously known as _token_) must be set in `SCW_SECRET_KEY` (`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`).
+~> **Important:** `SCALEWAY_ACCESS_KEY` was changed to `SCW_ACCESS_KEY`.
+This should be your access key (e.g. `SCWZFD9BPQ4TZ14SM1YS`).
+Your secret key (previously known as _token_) must be set in `SCW_SECRET_KEY` (`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`).
+
+Terraform can also read standard Scaleway configuration files.
+By doing so, you can use the same configuration between different tools such as the [CLI](https://github.com/scaleway/scaleway-cli) or [Packer](https://www.packer.io/docs/builders/scaleway).
 
 ## Resources
 
-All resources are from now on prefixed by `scaleway`, their product category and their product name (`scaleway_{product-category-name}_{product-name}_{resource-name}`). For instances an S3 bucket belongs to the `Storage` product category and is a resource of the `Object` product. Hence it is named: `scaleway_object_bucket`.
+All resources are from now on prefixed by `scaleway`, their product category and their product name (`scaleway_{product-category-name}_{product-name}_{resource-name}`).
+For instances an S3 bucket belongs to the `Storage` product category and is a resource of the `Object` product.
+Hence it is named: `scaleway_object_bucket`.
+
+### Migration guide for renamed resources
+
+Because the resources changed their name, we cannot use automatic state migration.
+We will first manually remove the resource from the terraform state and then use [`terraform import`](https://www.terraform.io/docs/import/usage.html) to import existing resources to a renamed resource.
+
+For instance, let's suppose that you have resource in `fr-par-1` such as:
+
+```hcl-terraform
+provider "scaleway" {
+    zone= "fr-par-1"
+}
+
+resource scaleway_server main {
+  name  = "foobar"
+  type  = "DEV1-S"
+  image = "cf44b8f5-77e2-42ed-8f1e-09ed5bb028fc"
+}
+```
+
+First, let's delete the resource from your terraform state using the [`terraform state`](https://www.terraform.io/docs/commands/state/index.html) command.
+You can do it using: `terraform state rm scaleway_server.main`.
+
+Once this is done, refactor your terraform code to:
+
+```hcl-terraform
+provider "scaleway" {
+    zone= "fr-par-1"
+}
+
+resource scaleway_instance_server main {
+  name  = "foobar"
+  type  = "DEV1-S"
+  image = "cf44b8f5-77e2-42ed-8f1e-09ed5bb028fc"
+}
+```
+
+and run `terraform import scaleway_instance_server.main fr-par-1/11111111-1111-1111-1111-111111111111` where `11111111-1111-1111-1111-111111111111` is the id of your resource.
+After importing, you can verify using `terraform apply` that you are in a desired state and that no changes need to be done.
```

Comment:
I think `terraform state mv scaleway_server.main scaleway_instance_server.main` is simpler than `state rm` then `import`. Isn't it?

Reply:
I've tried it but it does not work because the resources are from two different types.